vendor/github.com/skytap/skytap-sdk-go/skytap/interface.go

```diff
@@ -244,10 +248,106 @@ func (s *InterfacesServiceClient) Delete(ctx context.Context, environmentID stri
 		return err
 	}
 
-	_, err = s.client.do(ctx, req, nil)
+	_, err = s.client.do(ctx, req, nil, nil, nil)
```

Comment:
Deleting an interface will fail unless the vm is stopped. (The docs are wrong here -- I've filed DOC-2954 for correction.)